crates/primitives-traits/src/transaction/signed.rs

```diff
@@ -77,3 +82,23 @@ pub trait SignedTransaction:
     /// Fills [`TxEnv`] with an [`Address`] and transaction.
     fn fill_tx_env(&self, tx_env: &mut TxEnv, sender: Address);
 }
+
+impl<T: SignedTransaction> AlloyTransactionExt for T {
+    type Type = <T::Transaction as AlloyTransactionExt>::Type;
+
+    fn signature_hash(&self) -> B256 {
+        self.transaction().signature_hash()
+    }
+
+    fn is_dynamic_fee(&self) -> bool {
+        self.transaction().is_dynamic_fee()
+    }
+
+    fn effective_gas_price(&self, base_fee: Option<u64>) -> u128 {
+        self.transaction().effective_gas_price(base_fee)
+    }
```

Comment:
isn't this method equivalent to `alloy_consensus::Transaction::effective_tip_per_gas` ? @Rjected 
https://github.com/alloy-rs/alloy/blob/e747b1e863fe647ca45489ba59c867a47c937fdc/crates/consensus/src/transaction/mod.rs#L97-L117

Reply:
@mattsse do you know the answer to this?

Reply:
it's not equivalent, the tip is `min(maxFeePerGas - baseFee, maxPriorityFeePerGas)`

Reply:
are we interested in moving this into alloy too?